### ISSUE TITLE
Layout for BookShelf

### ISSUE THREAD

**Issue by Kjani01** [mobileappdevhm19/HINT-Reader#31]
While we are figuring out how to make the backend work, I started working on the frontend for the BookShelf. Right now it is just one dummy picture and I think our dummy should at least look a bit like the picture in Simons horizontal prototype.

I already started experimenting with pictures in tables and will continue tomorrow. If you want to see what I've done so far, you have to start the BookShelf in the Main method, not the BookShelfView. 

But I will show you on friday anyhow, I think I'm finished until then.

**Comment by Kjani01** [comment]
I have just commited the new BookShelf-Layout (but no worries, the old version still exists in another file 😄). 
It would be nice, if everyone tests, if the layout still produces overflows of the table or images. It was a bit difficult to adjust the size, because I found no way to adjust the table itself in dependency of the current screen (I will continue to look, though, maybe I just missed it today). I just adjust the image size now in dependency of the current screen size. 
But I only have two devices to test the screens 😄 

If the BookShelf now produces problems with the tests we already have or Maxi is just writing, then just go back to the previous version for our sprint review on friday, if I can't find another solution ;)

Have a nice weekend everybody!
